docs/src/languages/php/fpm.md

```diff
@@ -3,70 +3,107 @@ title: "PHP-FPM sizing"
+ - `70 MB` for the reserved memory.
 
- * `45 MB` for the average per-request memory
- * `70 MB` for the reserved memory
+These values allow most programs to run but you can adjust them to fit your needs.
 
-These are deliberately conservative values that should allow most programs to run without modification.
+To do so, in your `.platform.app.yaml` file, adjust the values of the `runtime.sizing_hints.reserved_memory` and `runtime.sizing_hints.request_memory` keys.
 
-You can change them by using the `runtime.sizing_hints.reserved_memory` and `runtime.sizing_hints.request_memory` in your `.platform.app.yaml`. For example, if your application consumes on average `110 MB` of memory for a request use:
+For example, if your application consumes on average `110 MB` of memory for a request, you can use:
 
-```yaml
+```yaml {location=".platform.app.yaml"}
 runtime:
     sizing_hints:
         request_memory: 110
 ```
 
-The `request_memory` has a lower limit of 10 MB while `reserved_memory` has a lower limit of 70 MB.
-Values lower than those are replaced with those minimums.
+Note that the minimum value for the `request_memory` key is 10 MB
+and the minimum value for the `reserved_memory` key is 70 MB.
+If you set lower values, they are automatically overridden with those minimums.
+
+For information on how to estimate the best `request_memory` value depending on your needs, 
+see how to [measure PHP worker memory usage](./fpm.md#measure-php-worker-memory-usage). 
 
-You can check the maximum number of PHP-FPM workers by opening an [SSH session](/development/ssh/_index.md) and running following command (example for PHP 7.x):
+To check the maximum number of PHP-FPM workers available to your site,
+open an [SSH session](/development/ssh/_index.md) and run the following command:
 
 ```bash
 grep -e '^pm.max_children' /etc/php/*/fpm/php-fpm.conf
+```
+
+You get output similar to the following,
+where workers are referred to as `children`:
+
+```bash
 pm.max_children = 2
 ```
 
-## Measuring PHP worker memory usage
+## Measure PHP worker memory usage
 
-To see how much memory your PHP worker processes are using, you can open an [SSH session](/development/ssh/_index.md) and look at the PHP access log:
+To see how much memory your PHP worker processes use, 
+open an [SSH session](/development/ssh/_index.md) 
+and look at the PHP access log:
 
 ```bash
 less /var/log/php.access.log
 ```
 
-In the fifth column, you'll see the peak memory usage that occurred while each request was handled. The peak usage will probably vary between requests, but to avoid the severe performance costs that come from swapping, your size hint should be somewhere between the average and worst case memory usages that you observe.
+In the fifth column of the output, 
+you can see the peak memory usage that occurred when each request was handled. 
+The optimal request memory is somewhere between the average and worst case memory usages listed.
```

Comment:
:lady_beetle: `somewhere between` seems rather vague. I'm not sure how useful it really is.